ckanext/scheming/templates/scheming/package/snippets/resources_list.html

```diff
@@ -9,40 +9,36 @@
   {% snippet "package/snippets/resources_list.html", pkg=pkg, resources=pkg.resources %}
 
 #}
-<section id="dataset-resources" class="resources">
-  <h3>{{ _('Data and Resources') }}</h3>
-  {% block resource_list %}
-    {% if resources %}
-      <ul class="{% block resource_list_class %}resource-list{% endblock %}">
-        {% block resource_list_inner %}
-          {% set can_edit = h.check_access('package_update', {'id':pkg.id }) and not is_activity_archive %}
-          {% for resource in resources %}
-            {% snippet 'package/snippets/resource_item.html', pkg=pkg, res=resource, can_edit=can_edit, is_activity_archive=is_activity_archive %}
-          {% endfor %}
-        {% endblock %}
-      </ul>
+
+{% set core_resources = h.get_core_resources(pkg) %}
+{% set can_edit = h.check_access('package_update', {'id':pkg.id }) and not is_activity_archive %}
+<section id="dataset-resources" class="resources core-resources">
+  <h3>{{ _('Core Resources') }}</h3>  
+  {% if core_resources %}
+    <ul class="resource-list">
+      {% for resource in core_resources %}
+        {% snippet 'package/snippets/resource_item.html', pkg=pkg, res=resource, can_edit=can_edit, is_activity_archive=is_activity_archive %}
+      {% endfor %}
+    </ul>
+  {% else %}
+    {% if h.check_access('resource_create', {'package_id': pkg['id']}) and not is_activity_archive %}
+        {% trans url=h.url_for(pkg.type ~ '_resource.new', id=pkg.name) %}
+          <p class="empty">This dataset has no core resources, <a href="{{ url }}">why not add some?</a></p>
```

Comment:
"why not add some" will be confusing as it is a url to add a general/extra resource. Adding core resources happens with help of "add links" next to missing resources.
I think we should remove the "why not add some" link from here.